packages/strapi/bin/strapi-new.js

```diff
@@ -65,7 +67,8 @@ module.exports = function (name, cliArguments) {
       },
       options: {
         authenticationDatabase: cliArguments.dbauth,
-        ssl: cliArguments.dbssl
```

Comment:
We don't have to store this information in the database file.
`options` object is send to the database hook for the connection and `force` is not an available key.
And the `force` argument is just here to display or not the warning message, we never will use it after the app generation.

So I think we have to remove updates of this file.

Reply:
I agree, where do you think I should add this value so that it's accessible in `connectivity`. `scope.dbforce` maybe?